runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -10329,6 +10329,505 @@ inlineCompareAndSwapNative(
    }
 
 
+/**
+ * \brief
+ *   Generate inlined instructions equivalent to java/lang/StringCoding.countPositives
+ *
+ * \param node
+ *   The tree node
+ *
+ * \param cg
+ *   The Code Generator
+ */
+static TR::Register* inlineCountPositives(TR::Node* node, TR::CodeGenerator* cg)
+   {
+   // Arguments to countPositives
+   // Byte array
+   auto ba = cg->evaluate(node->getChild(0));
```

Comment:
I will let you process and action all of Brad's comments before reviewing this thoroughly, but one thing I will mention right away are my strong opinions on the use of the `auto` keyword and its impact on future code understanding for anyone but the author.  I may be known to let one or two slip in on occasion, but since you have many (12) in succession I'll ask that you specify the data type for each (they look to be either `TR::Node *`'s or `TR::Register *`'s in your case.

I'll echo Brad's comments about variable naming as well.  It is easier to read the code later if the variables that contain nodes are suffixed with `Node` and registers are suffixed with `Reg`.